### DISCUSSION TITLE
Lora > lyco handler change question

### DISCUSSION THREAD

**Discussion by brknsoul** [vladmandic/sdnext#1349] [Q&A] [answered]
If I choose Lycora to handle my LoRAs, do I need to move the LoRAs out of the `lora` folder, or will Lycora find them automatically?

**Comment by vladmandic** [comment]
honestly, i don't see the need to have them separated to start with so i have them set to same folder in settings -> system paths.

**Comment by lbeltrame** [comment]
As far as I could tell, if you set lyco to cover also lora, you won't be able to see the LoRA paths, only the LyCORIS ones. At least that's what happened a few days ago, but I didn't check whether it was an actual bug or not.

**Comment by Davikar** [comment]
If I chose "Use LyCoris handler for all Lora types" and restart, it will hide the Lora tab and only show Lycoris in extra networks. If I open the Lycoris tab it will show only Loras the first time and if I reload it, it will only show Lycoris.

**Reply by brknsoul** [reply, marked as answer]
Looks like I have to either copy/move the loras to the lycoris folder, or vice-versa and change the lycoris folder in settings.

Perhaps the lycoris handler should enumerate both folders upon server start/refresh.

**Reply by vladmandic** [reply]
Once it's out of experimental stage, sure.

**Reply by PyroclasticFloe** [reply]
On my windows system, I have a symbolic link pointing from the lora folder to the lycoris folder, largely due to the fact that the Civitai Helper extension doesn't check the lycoris folder and won't show information for anything under lycoris. Since the author of that extension has indicated that they won't support other extensions, would it be possible to make hiding the lora folder an option?
